unicodetools/data/ucd/dev/VerticalOrientation.txt

```diff
@@ -2189,6 +2189,7 @@ FFFC..FFFD     ; U  # So     [2] OBJECT REPLACEMENT CHARACTER..REPLACEMENT CHARA
 1BC9F          ; R  # Po         DUPLOYAN PUNCTUATION CHINOOK FULL STOP
 1BCA0..1BCA3   ; R  # Cf     [4] SHORTHAND FORMAT LETTER OVERLAP..SHORTHAND FORMAT UP STEP
 1CEB0..1CEB3   ; R  # So     [4] HORIZONTAL ZIGZAG LINE..BLACK RIGHT TRIANGLE CARET
+1CEE0..1CEEF   ; R  # So    [16] GEOMANTIC FIGURE POPULUS..GEOMANTIC FIGURE VIA
```

Comment:
As noted in PR #590 and in https://github.com/unicode-org/sah/issues/303 I think these symbols should get vo=U, and the whole new block should default to that.